couchbase-driver/src/test/java/org/jnosql/diana/couchbase/configuration/CouchbaseContainer.java

```diff
@@ -0,0 +1,62 @@
+/*
+ *  Copyright (c) 2017 Otávio Santana and others
+ *   All rights reserved. This program and the accompanying materials
+ *   are made available under the terms of the Eclipse Public License v1.0
+ *   and Apache License v2.0 which accompanies this distribution.
+ *   The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ *   and the Apache License v2.0 is available at http://www.opensource.org/licenses/apache2.0.php.
+ *
+ *   You may elect to redistribute this code under either of these licenses.
+ *
+ *   Contributors:
+ *
+ *   Otavio Santana
+ */
+package org.jnosql.diana.couchbase.configuration;
+
+import org.testcontainers.containers.FixedHostPortGenericContainer;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+
+public class CouchbaseContainer {
```

Comment:
FYI there is `CouchbaseContainer` in Testcontainers

Reply:
Oh sorry. I'll check that out. Thanks!

Reply:
I tried using testcontainers, but it was not possible due to a problem with Fts. Jnosql uses the FullTextIndex service. I think about closing this PR and continuing the discussion on testcontainers project. What do you think?